modules/ROOT/pages/availability_scaling/index.adoc

```diff
@@ -0,0 +1,140 @@
+
+Summary::
+As there are many different ways to solve a required scenario, ownCloud support _cant_ give advice for a particular solution that fits your needs but may be able to help to get the required oCIS component ready to run.
++
+For availability, storing user files is by nature mandatory to be on xref:prerequisites/index.html#filesystems-and-external-storage[shared storage] to be accessible by the nodes and/or services.
+
+==== Classic High Availability
+
+When using the classic form of high availability (HA), you can either make a setup where the nodes are both active (active-active) or one node is spare, waiting for a failure to occur (active-passive). With an A-A setup, a loadbalancer is needed in front of the nodes and in case of a failure, the remaining node has to take all the load additionally the failed one served. 
+
+image::availability_scaling/a_a_ha_with_data_server.svg[width=310]
+
+Summary::
+This usecase can be considered if hardware availability is the primary obective. When using an active-active configuration, each node is addressed by a loadbalancer for load distribution. This requires that the nodes have the same setup and services are bound to the nodes. *Scalability is hardware dependent and in case of an active-passive setup, you can even get reduced scalability*.
+
+==== Clustering
+
+The main objective for clustering is not only availability but also distributing load across _multiple_ nodes. Clustering mandatory needs a loadbalancer (LB).
+
+image::availability_scaling/a_a_cluster_with_data_server.svg[width=450]
+
+Summary::
+Clustering provides better availability and scalability for growing loads and covers failover if a node fails but is still very hardware focused. A cluster environment can grow very complex with many dependencies, see the section xref:availability[Availability]. *Scalability is much better but lacks when it comes to load based dynamic assignment of services*.
+
+== Scalability
+
+There are two classical ways to achieve scalability, which is Scale-Up and Scale-Out.
+
+[width="100%",cols="^100%,^100%",frame=none,grid=none]
+|===
+2+| The pictures below show the different ways of classical scaling
+
+a| image::availability_scaling/scale_up.svg[width=214]
+a| image::availability_scaling/scale_out.svg[width=300]
+|===
+
+What may sound simple with regards to a _service_, services can be a complex topic in reality as they may contain a lot of software components and their configuration building it. Adding more services or migrating a service can therefor be a challenging task adding complexity and can introduce sources of error.
+
+Summary::
+Scalability and availability are often aligned to each other and the decision how to achieve the goal set can be a complex task. This becomes even more true when dynamic load balancing comes into play. Because services consist in real life of many components to take care on, real dynamic adoption and dynamic migration is hard to be achieved.
```

Comment:
```suggestion
Scalability and availability are often aligned to each other and the decision how to achieve the goal set can be a complex task. This becomes even more true when dynamic load balancing comes into play. Because services consist of many components to take care of, real dynamic adoption and dynamic migration is hard to be achieved.
```